DATA/production/calib/its-threshold-aggregator.sh

```diff
@@ -18,7 +18,7 @@ ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR;Na
 PROXY_INSPEC="tunestring:ITS/TSTR/0;runtype:ITS/RUNT/0;fittype:ITS/FITT/0;scantype:ITS/SCANT/0;eos:***/INFORMATION"
 
 WORKFLOW="o2-dpl-raw-proxy $ARGS_ALL --proxy-name its-thr-input-proxy --dataspec \"$PROXY_INSPEC\" --network-interface ib0 --channel-config \"name=its-thr-input-proxy,method=bind,type=pull,rateLogging=0,transport=zeromq\" | "
-WORKFLOW+="o2-its-threshold-aggregator-workflow -b --ccdb-url=\"http://alio2-cr1-flp199.cern.ch:8083\" $ARGS_ALL | "
+WORKFLOW+="o2-its-threshold-aggregator-workflow -b --ccdb-url=\"\" $ARGS_ALL | "
```

Comment:
Hi Ivan, 
Why have you removed the CCDB?

Reply:
to use the o2-ccdb-populator workflow. In o2-its-threshold-aggregator-worfklow, if I leave the ccdb-url empty, it does not use the ccdb API but ships the DCSConfigObject to the ccdb-populator wf.

Reply:
Hi Ivan.
Ok, I see. But honestly, I don't like much that you use an empty argument, wouldn't it be clearer to pass something like "not-used" that you then parse in the task?

Reply:
@chiarazampolli for the task itself it doesn't matter because I have an if condition like that:
```
if (!mCcdbUrl.empty()) {
  //do stuff
}
```
I forgot and actually I can completely remove the --ccdb-url "" from this PR because the default value it's already an empty string. So if you agree I would simply do that.

Reply:
Ciao @iravasen , 

Yes, please, remove it as it is misleading. 

Thanks!

Reply:
done